github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits-0.49.yaml

```diff
@@ -1867,7 +1867,6 @@ presubmits:
       preset-dind-enabled: "true"
       preset-docker-mirror-proxy: "true"
     name: pull-kubevirt-e2e-k8s-1.22-sig-monitoring-0.49
-    optional: true
```

Comment:
Do we really want to do this for the 0.49 lane, not sure we have enough data for this? Or is this required as we are going to backport some stuff here?

Reply:
@dhiller Currently we don't have any PR related to monitoring that has to be backported. I was only thinking about a possibility for that to happen. I think that in general if that will happen and we will keep it optional in 0.49, it will be tested on main but not tested on the 0.49 which is inconsistent, WDYT?

Reply:
If you remove the `optional: true` it will still be tested but the merge will not be gated by tide should the lane fail.

Reply:
@dhiller I see, and since the report will appear, the reviewer will notice the failure of the lane. So lets keep optional then. Thank you for the brainstorm!